mods/ccore/notify.lua

```diff
@@ -0,0 +1,46 @@
+local notify_ids = {}
+
+-- Support HUD notifications under weildhand with timeout. Leave timeout nil to use default of 12 seconds
+ccore._notify = function(playername, message, timeout)
+    assert(timeout == nil or timeout > 0, "Invalid value: timeout must be a number > 0 or leave nil for default")
+    local player = minetest.get_player_by_name(playername)
+    -- TODO: Implement queue of notifications to manage smooth transitioning
+    -- avoid missed notifications due to being replaced too quickly
+    -- For now replace current notification
+    if notify_ids[playername] ~= nil then
+        local id = notify_ids[playername]
+        player:hud_remove(id)
+    end
+    -- Create and store HUD
+    local id = player:hud_add({
+        hud_elem_type = "text",
+		name = "Wield",
```

Comment:
hud name is mostly redundant
but if there's a need to name it, using the normal naming format of `<modname>:<name>` is preferable to prevent accidental overrides.